<launch>
    <include file="$(find infinity_uwb_ros)/launch/uwb_node_win10.launch">
    </include>
    <node pkg="swarm_drone_proxy" name="swarm_drone_proxy" type="swarm_drone_proxy_node" output="screen"> 
    </node>
    <node pkg="swarm_vo_fuse" name="swarm_vo_fuse" type="swarm_vo_fuse_node" output="screen">
        <param name="max_keyframe_num" value="20" type="int" />
    </node>
    <node pkg="swarm_vo_fuse" name="vo_data_gen" type="test_vo_datagen.py" output="screen"> 
        <param name="drone_num" value="10" type="int" />
        <param name="self_id" value="0" type="int" />
        <param name="enable_pub_drone_source" value="false" type="bool" />
    </node>
</launch>